### ISSUE TITLE
Poor preprocessing result on EMPIAR-10005 using CryoSPARC leads to non-decreasing training loss in CryoNeFEN

### ISSUE THREAD

**Issue by liuzhen0212** [yuehuang2023/cryoNeFEN#25]
Hi,

I am trying to reproduce the CryoNeFEN training pipeline using the EMPIAR-10005 dataset. However, the preprocessing result from CryoSPARC appears to be very noisy and the CryoNeFEN training loss does not decrease during optimization.

I would like to ask whether this behavior is expected or if there might be something wrong with my preprocessing or training setup.

**Dataset**
<img width="457" height="270" alt="Image" src="https://github.com/user-attachments/assets/33bed423-32e5-4f63-b5a9-711d6a78990c" />

**CryoSPARC preprocessing pipeline**
<img width="1227" height="517" alt="Image" src="https://github.com/user-attachments/assets/01f91192-a105-492c-9ecf-ce4798a31804" />

**CryoNeFEN training**
<img width="1079" height="713" alt="Image" src="https://github.com/user-attachments/assets/dbd7edef-470a-4aec-a0cf-4e176afa6e0c" />

**Questions**

Is the CryoSPARC preprocessing result expected for EMPIAR-10005?

Could the noisy reconstruction cause CryoNeFEN training to stall?

Are there recommended preprocessing steps or parameters for this dataset?

Any suggestions would be greatly appreciated.
Thank you!
